vm/rust/src/lib.rs

```diff
@@ -338,6 +355,18 @@ pub extern "C" fn cairoVMExecute(
                 let actual_fee = t.actual_fee.0.into();
                 let data_gas_consumed = t.da_gas.l1_data_gas.into();
 
+                // write trace to file
+                #[cfg(feature = "tracing")]
+                {
+                    let duration = t.duration.unwrap();
+                    writer
+                        .write_all(
+                            format!("{},{}\n", txn_and_query_bit.txn_hash, duration.as_nanos())
```

Comment:
Although the duration is made available in blockifier, txn hash is not available there, so I had to do the logging here in juno